packages/web3torrent/src/components/torrent-info/peer-speed-info/PeerSpeedInfo.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import prettier from 'prettier-bytes';
+import './PeerSpeedInfo.scss';
+import {TorrentUI, IdleStatuses} from '../../../types';
+
+export type PeerSpeedInfoProps = {torrent: TorrentUI};
+
+const PeerSpeedInfo: React.FC<PeerSpeedInfoProps> = ({torrent}: PeerSpeedInfoProps) =>
```

Comment:
Do you think this might be more clear as `PeerNetworkStats` or something like that?

Reply:
I suck at naming stuff, that IS a more descriptive name

Reply:
https://github.com/statechannels/monorepo/pull/1754/commits/509184812bccb4e4bbb5533dbf84feb8d9339b48